pkg/gui/services/custom_commands/handler_creator.go

```diff
@@ -292,7 +292,17 @@ func (self *HandlerCreator) finalHandler(customCommand config.CustomCommand, ses
 			if strings.TrimSpace(output) == "" {
 				output = self.c.Tr.EmptyOutput
 			}
-			return self.c.Alert(cmdStr, output)
+
+			var title string
+			if customCommand.OutputTitle != "" {
+				title, err = resolveTemplate(customCommand.OutputTitle)
+				if err != nil {
+					return err
+				}
+			} else {
+				title = cmdStr
```

Comment:
We could avoid having the `else` branch here by just assigning the `cmdStr` value to `title` as a default value and letting the conditional to override it if `customCommand.OutputTitle != ""`. What do you think?

Reply:
Sure, why not. Changed in ba35802e7852.